src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -8,360 +8,710 @@ authors:
   - mustafakurtuldu
   - thomassteiner
 date: 2018-02-19
-updated: 2020-05-12
+updated: 2020-08-05
 hero: hero.jpg
 tags:
   - blog
   - fonts
 ---
+<!--
 
-Variable fonts can significantly reduce the size of your font files and make
-it possible to animate your font characters. This guide explains how
-variable fonts work, how typographers implement variable fonts, and how
-to work with variable fonts in CSS.
+TODO add these to "authors" in yaml ↑
+{ % include "web/_shared/contributors/dcrossland.html" %}
+{ % include "web/_shared/contributors/roeln.html" %}
 
-## Browser compatibility {: #compatibility }
+-->
 
-As of May 2020 variable fonts are supported in most browsers. See
-[Can I use variable fonts?](https://caniuse.com/#feat=variable-fonts)
-and [Fallbacks](#fallbacks).
+In this article, we will look at what variable fonts are, the benefits they
+offer, and how we can use them in our work. First, let's review how typography
+works on the web, and what innovations variable fonts bring.
 
 ## Introduction
 
-The terms font and typeface are used interchangeably in the developer industry.
-However there is a difference: A typeface includes an entire family of designs,
-such as the [Roboto](https://fonts.google.com/specimen/Roboto) typeface.
-Meanwhile a font is one of the digital files of that family, like "Roboto Bold"
-or "Roboto Italic." In other words, a typeface is what you _see_, and the font is
-what you _use_.
+The terms font and typeface are often used interchangeably by developers.
+However there is a difference: A typeface is the underlying visual design that
+can exist in many different typesetting technologies, and a font is one of these
+implementations, in a digital file format. In other words, a typeface is what
+you _see_, and the font is what you _use_.
+
+Another concept that is often overlooked is the distinction between a style, and
+a family. A style is a single and specific typeface, such as Bold Italic, and a
+family is the complete set of styles.
+
+Before variable fonts, each style was implemented as a separate font file. With
+variable fonts, all styles can be contained in a single file.
 
 <figure class="w-figure">
-  <img src="robot-specimen.png" alt="A specimen of Roboto">
-  <figcaption class="w-figcaption">
-    Examples of the Roboto typeface that was designed and developed
-    Christian Robertson.
+  <img src="roboto-family-specimen.png" style="max-width: 100%;" alt="A specimen composition and list of different styles of the Roboto family">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Above: Left, a specimen of the Roboto typeface family. Right, named styles within the family.
```

Comment:
```suggestion
    Left: a specimen of the Roboto typeface family. Right: named styles within the family.
```
I don't think `Above` is necessary because it's pretty clearly styled as the caption for the image